src/setuptools_pyproject_migration/__init__.py

```diff
@@ -71,8 +82,17 @@ def _generate_entry_points(entry_points: Optional[Dict[str, List[str]]]) -> Dict
 
     parsed_entry_points: Dict[str, Dict[str, str]] = {}
 
-    for eptype, raweps in entry_points.items():
-        parsed_entry_points[eptype] = dict(map(_parse_entry_point, raweps))
+    if isinstance(entry_points, str):
+        # INI-style
+        parser = configparser.ConfigParser()
+        parser.read_string(entry_points)
+        for eptype, section in parser.items():
+            if eptype != "DEFAULT":
```

Comment:
I wonder what `setuptools` would do if it gets passed an entrypoints specification with a `DEFAULT` section? If it doesn't just crash or fail, that would probably be worth adding a test case for.

Reply:
Since I was doing some tests anyway, I tried it out: it turns out `pkg_resources.EntryPoint.parse_map()` will happily accept a section named `DEFAULT` and create a corresponding entry point group, which probably suggests that that's what setuptools will do as well. I didn't actually create a project to test it, but I think this is definitely worth having a test case for.

Reply:
Yeah, I assumed that the `DEFAULT` group would be pretty meaningless, but then again, this would not be the first time such assumptions bit us.

I think it's supported by accident rather than by design.

We could remove that `if` block… actually I'd probably replace the condition with a check to see if the resulting group is empty so we don't needlessly create empty groups.

Reply:
Yeah checking for empty groups sounds more practical.